buildlib/pr/main.yml

```diff
@@ -269,33 +269,38 @@ stages:
         name: althca
         demands: ucx_althca -equals yes
         test_perf: 0
+        proto_enable: yes
```

Comment:
why need to set "proto_enable: yes" for every case?

Reply:
to be less error prone (and make sure we explicitly set it to something which is not `no`)

Reply:
maybe set it once in high level?